products/oracle-database.md

```diff
@@ -0,0 +1,124 @@
+---
+title: Oracle Database
+category: db
+iconSlug: oracle
+permalink: /oracle-database
+versionCommand: show version
+releasePolicyLink: https://www.oracle.com/us/assets/lifetime-support-technology-069183.pdf
+releaseDateColumn: true
+activeSupportColumn: Premier Support
+eolColumn: Extended Support
+
+releases:
+
+-   releaseCycle: "21c"
+    lts: false
+    latest: "21.4.2.0.0"
+    support: 2024-04-30
+    eol: false
+    latestReleaseDate: 2022-12-16
```

Comment:
What were your source(s) for the dates ? I ask this so I can quickly validate. Do not hesitate to put, for each release, a link to the page that give the information.

Moreover, you should clarify what's being tracked here. According to https://support.oracle.com/knowledge/Oracle%20Database%20Products/742060_1.html there are _On-Premises Server Releases_ and _Oracle Public Cloud Releases_, and those releases have different release dates for the same major versions.

Reply:
https://github.com/endoflife-date/endoflife.date/issues/2127#issuecomment-1422069730

Reply:
https://github.com/endoflife-date/endoflife.date/issues/2127#issuecomment-1422070186

Reply:
- https://support.oracle.com/knowledge/Oracle%20Cloud/21202110_9.html
- https://support.oracle.com/knowledge/Oracle%20Cloud/2796590_1.html

Reply:
Also from [Wikipedia](https://en.wikipedia.org/wiki/Oracle_Database) : 

![image](https://user-images.githubusercontent.com/5235127/218881980-00c57cf0-5c01-4214-aa04-f68d051ba499.png)

Reply:
Also from[ Oracle Database 21c Release Update & Release Update Revision July 2022 Known Issues (Doc ID 21202207.9)](https://support.oracle.com/knowledge/Oracle%20Database%20Products/21202207_9.html)